db/migrations/sqls/20201201100654-add-new-grants-up.sql

```diff
@@ -0,0 +1,4 @@
+alter table application_assessment drop column grant_amount_awarded;
+alter table application_assessment add column lrsg_closed_businesses_amount decimal(12,2) default 0;
+alter table application_assessment add column lrsg_sector_amount decimal(12,2) default 0;
+alter table application_assessment add column lrsg_open_amount decimal(12,2) default 0;
```

Comment:
Why do we have a decimal precision of 12 here?

Reply:
Just so we have wiggle-room to use larger numbers if necessary

Reply:
That is fair, although I do feel 12 is large precision, but it doesn't matter too much here.